static/js/screenly-ose.coffee

```diff
@@ -413,6 +420,7 @@ API.View.AssetRowView = class AssetRowView extends Backbone.View
     (@$ ".toggle input").prop "checked", @model.get 'is_enabled'
     (@$ ".asset-icon").addClass switch @model.get "mimetype"
       when "video"   then "icon-facetime-video"
+      when "streaming"   then "icon-facetime-video"
```

Comment:
The reason there are x spaces before "then" is that someone wanted to align all the `then`s vertically. So either align 'em all or don't align them all, but either way this`then` won't have more than 1 space before it.